src/classes/RD_RecurringDonations.cls

```diff
@@ -91,7 +91,8 @@ public class RD_RecurringDonations {
 
     /*******************************************************************************************************
     * @description holds the current custom field mappings used when creating opps for the RD.
-    */ 
```

Comment:
Does the comment block at the top of this class that explains the various behaviors need to be updated?

Reply:
Yes absolutely.